include/boost/math/distributions/dirichlet.hpp

```diff
@@ -200,16 +200,14 @@ class dirichlet_distribution
 
   // Get the concentration parameters.
   const RandomAccessContainer &get_alpha() const { return m_alpha; }
-  // Set the concentration parameters.
-  RandomAccessContainer &set_alpha() { return m_alpha; }
 
   // Get the order of concentration parameters.
   size_t order() const { return m_alpha.size(); }
 
   // Get alpha from mean and variance.
   void find_alpha(
-      RandomAccessContainer &&mean,     // Expected value of mean.
-      RandomAccessContainer &&variance) // Expected value of variance.
+      RandomAccessContainer &mean,     // Expected value of mean.
+      RandomAccessContainer &variance) const// Expected value of variance.
```

Comment:
```const``` here needs to be removed. don't know what i was experimenting.

Reply:
Why does `const` need to be removed here?

Reply:
Because m_alpha is being assigned in this function but ```const``` forces it to be read-only.